applications/CoSimulationApplication/python_scripts/solver_wrappers/kratos/kratos_base_wrapper.py

```diff
@@ -98,8 +102,25 @@ def _CreateAnalysisStage(self):
 
     def __GetAnalysisStage(self):
         if self.settings["solver_wrapper_settings"].Has("analysis_stage_module"):
-            analysis_stage_module = import_module(self.settings["solver_wrapper_settings"]["analysis_stage_module"].GetString())
-            return analysis_stage_module.Create(self.model, self.project_parameters)
+            module_name = self.settings["solver_wrapper_settings"]["analysis_stage_module"].GetString()
+            analysis_stage_module = import_module(module_name)
+            if hasattr(analysis_stage_module, "Create"):
+                return analysis_stage_module.Create(self.model, self.project_parameters)
+            else:
+                KM.Logger.PrintWarning("KratosBaseWrapper", f'The analysis_stage_module "{module_name}" does not have a "Create" function, trying to create the AnalysisStage directly...')
+                if self.settings["solver_wrapper_settings"].Has("analysis_name"):
+                    analysis_stage_name = self.settings["solver_wrapper_settings"]["analysis_name"].GetString()
+                else:
```

Comment:
I think it should be in the other way around. You assume the name is the module name in UpperCamelCase. If not you search for a custom one. This is the same we assume in the standard MainKratos.py.

Reply:
I don't understand, what should I check?

Reply:
You do a try with what you put in the else. If doesn't succeed, you try to parse a custom analysis name.

Reply:
But I think it more consistent the other way around, if my name is not "standard" I will provide it manually, otherwise I will be doing two checks instead of one

Reply:
Okay....

Reply:
Huuuuuum.....